templates/shared_services/sonatype-nexus-vm/terraform/data.tf

```diff
@@ -47,3 +47,8 @@ data "azurerm_private_dns_zone" "nexus" {
   name                = "nexus-${var.tre_id}.${data.azurerm_resource_group.rg.location}.cloudapp.azure.com"
   resource_group_name = local.core_resource_group_name
 }
+
+data "azurerm_container_registry" "mgmt_acr" {
```

Comment:
I can't see where this is being used, might be I've overlooked something.

Reply:
Thank you, this is left over from a previous test. I have removed it now